app/database/db_build.sql

```diff
@@ -11,6 +11,9 @@ CREATE TABLE nuggets(
   description TEXT,
   img_url VARCHAR(100),
   author VARCHAR(30) NOT NULL
-)
+);
 
--- COMMIT;
+INSERT INTO nuggets (lat,long,category,title,description,img_url,author)
+VALUES (100,100,'food','bigwave','tsunami',null,'me');
```

Comment:
should have spaces after commas.

also should put it more realistic values for lat and long, e.g. -30.234903